bash.exe.stackdump

```diff
@@ -0,0 +1,9 @@
+Stack trace:
```

Comment:
This surely should not be committed, should it?

Reply:
Yes it should not be. Deleted the file thanks :)